test-data/unit/fixtures/f_string.pyi

```diff
@@ -0,0 +1,41 @@
+# Builtins stub used for format-string-related test cases.
+# We need str and list, and str needs join and format methods.
+
+from typing import TypeVar, Generic, builtinclass, Iterable, Iterator, List, overload
+
+T = TypeVar('T')
+
+@builtinclass
```

Comment:
I don't know, I think nothing -- maybe it's a relic? I just copied that code from fixtures/list.pyi. I've removed it.